helpers/article.js

```diff
@@ -205,9 +212,60 @@ class ArticleHelper {
         as: 'author',
         attributes: ['username', 'bio', 'image']
       }],
-      attributes: ['slug', 'title', 'description', 'body', 'createdAt', 'updatedAt']
+      attributes: ['id', 'slug', 'title', 'description', 'body', 'createdAt', 'updatedAt']
     });
     return article;
   }
+
+  /**
+  * @param  {object} req - Request object
+  * @returns {object} response
+  *  @static
+  */
+  static async createArticleTag(req) {
+    const createdTag = await Tag.create({
+      name: req.body.name,
+      articleId: req.body.articleId
+    });
+    return createdTag.dataValues;
+  }
+
+  /**
+   * @returns {object} response
+   *  @static
+  */
+  static async listTags() {
+    const tagList = await Tag.aggregate('name', 'DISTINCT', { plain: false }).map(row => row.DISTINCT);
+    return tagList || [];
+  }
+
+  /**
+   * Check if the same tag was added before
+   * @param  {object} req - Request object
+   * @param {object} res - Response object
+   * @param {function} next - calls the next middleware
+   * @return {json} Returns json object
+   * @static
+   */
+  static async isNotTagAdded(req, res, next) {
```

Comment:
What is the purpose of this function?

Reply:
> What is the purpose of this function?

It is for checking if Tag was added to the article before